App/TopTrumps/TopTrumps.Data.Tests/Services/CardsServicesTests.cs

```diff
@@ -64,7 +64,7 @@ public async Task GetAllCards_ShouldReturnAllCardsInTheDatabase()
             const string sql = "SELECT";
             var expected = new List<Card> { new Card() };
 
-            _helper.Setup(x => x.GetAllCardsQuery())
```

Comment:
Why are we renaming the `GetAllCardsQuery` to `GetAllUsersQuery` ?

Reply:
fixed. didn't mean to rename